MODULE.bazel

```diff
@@ -7,13 +7,14 @@ module(
     compatibility_level = 1,
 )
 
+# TODO(#203): remove dependency on compiling C++ tools
```

Comment:
How is this TODO supposed to be resolved? Removing rules_cc and the tools written in C++ from rules_proto?

Reply:
yes, or else publish pre-built binaries on the GH releases

Reply:
Either option doesn’t seem very inviting. First one is incompatible and second one support only prebuilt platforms and requires some nontrivial amount of work.

None of those options seem necessary to do, we can also keep rules_cc dependency, so I’d propose that this todo is removed.